tests/unit/common/spatial/test_point.py

```diff
@@ -16,37 +16,37 @@
 # limitations under the License.
 
 
-from unittest import TestCase
+import pytest
 
 from neo4j._spatial import (
     Point,
     point_type,
 )
 
 
-class PointTestCase(TestCase):
+class PointTestCase:
 
     def test_wrong_type_arguments(self):
         for argument in (("a", "b"), ({"x": 1.0, "y": 2.0})):
-            with self.subTest():
-                with self.assertRaises(ValueError):
+            with pytest.subtests():
+                with pytest.raises(ValueError):
                     Point(argument)
```

Comment:
Instead of `unittest.subTest`, I recommend to rely on pytest's parametrization feature. Something like this

```python
    @pytest.mark.parametrize("argument", ("a", "b"), ({"x": 1.0, "y": 2.0}))
    def test_wrong_type_arguments(self, argument):
        with pytest.raises(ValueError):
            Point(argument)
```